core/evtforward/ethereum/filterer.go

```diff
@@ -108,7 +108,7 @@ func NewLogFilterer(
 
 	collateralBridgeFilterer, err := bridge.NewErc20BridgeLogicRestrictedFilterer(collateralBridge.Address(), ethClient)
 	if err != nil {
-		return nil, fmt.Errorf("couldn't create log filterer for ERC20 brigde: %w", err)
+		return nil, fmt.Errorf("couldn't create log filterer for ERC20 bridge: %w", err)
```

Comment:
Filterer>filter

Reply:
nope filterer is actually the right term here.